apps/omg_watcher_rpc/test/omg_watcher_rpc/web/controllers/transaction_test.exs

```diff
@@ -29,11 +31,11 @@ defmodule OMG.WatcherRPC.Web.Controller.TransactionTest do
 
   @eth OMG.Eth.RootChain.eth_pseudo_address()
   @other_token <<127::160>>
-  @eth_hex @eth |> Encoding.to_hex()
-  @other_token_hex @other_token |> Encoding.to_hex()
+  @eth_hex Encoding.to_hex(@eth)
+  @other_token_hex Encoding.to_hex(@other_token)
   @default_data_paging %{"limit" => 200, "page" => 1}
 
-  describe "getting transaction by id" do
+  describe "/transaction.get" do
```

Comment:
don't want to throw this at you for this PR, but we could standardize on this `describe` convention in the entire file, when you're at it. I looked and it seems straightforward, maybe 2 `describe` sections into one, since they share an endpoint.

Reply:
Would this work? https://github.com/omisego/elixir-omg/pull/1224/commits/75207a787f3c6866d3089a66d7a0caea5940136f